docs/low-code-sql/data-tests/use-project-tests.md

```diff
@@ -0,0 +1,141 @@
+
+### Run a test
+
+After you’ve developed your project test and executed all of your models, you can run the test.
+
+1. Click the **Play** button on either the canvas or an individual Gem to execute your test. The table input to the data test Gem is what’s tested.
+
+   1. Run a whole test. Clicking the canvas **Play** button executes the complete test and shows the interim data after the Data Test Gem. Additionally, it displays the final test status, succeeded, warning, or failed, in a summary.
+
+   2. Run a partial test. Clicking the Gem **Play** button executes the test SQL up to a particular Gem, and displays that Gem’s data on the output. This option doesn’t execute the data test.
+
+2. Click **See Run Details** to view the test Summary. Depending on the outcome of the test, the icon displays a different color.
+
+   ![See Run Details](img/project-test-run-details.png)
+
+   You can click to expand the test logs in the Summary to view the dbt logs.
+
+   ![View test summary](img/project-test-summary.png)
+
+In addition to the previous relational integrity test, you can create tests to check that the total payment amount from customers is positive or simply check that all of your tables are still accessible. You can test any series of transformation Gems, because a project test simply asserts that the resulting table meets a certain criteria.
+
+## Fix a failed test
+
+If your project test fails, check your test model for any changes.
+
+![Failed test](img/project-test-fail.png)
+
+- Click the test output to see your failed rows.
+
+Some examples of test failure causes include:
+
+- Happy path: There is now a row in the table → you have detected the thing you were trying to detect!
+- The table no longer exists since one or more input data sources were deleted or inaccessible
+- Dataset schema changes, such as in the column or datatype
+- The seed in the test is different from what’s in your warehouse
+- Code releases that caused a breaking change
+
+:::note
+
+Make sure you have write permission to create a new table in your data warehouse, otherwise you may run into errors while trying to run your own tests.
+
+:::
+
+By default, project tests are configured to fail if the table has one or more rows. You can also modify the passing condition of your test through advanced options.
+
+## Configure a test
+
+You can configure your own passing condition of your project test.
+
+You can use the configurations to help decide which cases to focus on if you have too many failed rows. Or conversely, you might need to pay more attention as the number of rows in a table increases. For example, you might have a customer churn use case where many customers have a few comments (represented by rows in a table), but you need to change course when the number of customer comments exceeds a threshold of five comments. Configuring your test can help you identify and take action, such as assigning more resources to assist that customer.
```

Comment:
I suggest deleting this...

"You can use the configurations to help decide which cases to focus on if you have too many failed rows. Or conversely,"

...because before and after "conversely" are presenting the same idea.

Reply:
Done.